_code/EMBEDDING.md

```diff
@@ -61,20 +62,47 @@ to the documentation files using the tool. The most important points here are:
    ./embed-code-macos -config-path="config-v1.yml" -mode="embed"
    ```
 
-### How to add a new code snippet?
+### Adding a new example project
 
-1. Add a new snippet in the appropriate repository.
-2. Make sure it builds successfully: `./gradlew build`.
-3. Go to the `SpineEventEngine/documentation` project and insert code 
-   embedding directives where needed.
-4. Navigate to the `_code` directory.
-5. Execute: `./embed-code -config-path="config-of-your-choice.yml" -mode="embed"`.
+1. Make sure the project you're going to add has a top-level `buildAll` Gradle task.
+
+   See the [build of the Hello Example](https://github.com/spine-examples/hello/blob/master/build.gradle)
+   for the declaration of such a task. This task must present in both single- 
+   and multi-module Gradle example projects that are going to be used for 
+   embedding into this site.
+
+   If you are interested in the details on why the `buildAll` task is needed, 
```

Comment:
Let's simplify this whole paragraph.

"See the declaration of `buildAll` task for more details."